NEWS.md

```diff
@@ -1,3 +1,13 @@
+Julia v0.6.0 Release Notes
+==========================
+
+Breaking changes
+----------------
+
+This section lists changes that do not have deprecation warnings.
+
+  * Operations between `Float16` and `Integers` now return `Float16` instead of `Float32`. ([#17261])
+
```

Comment:
Yeah I think this makes sense. Maybe copy all the headings but leave the rest of them empty for now?